packages/jest-circus/src/__tests__/__snapshots__/after_all-test.js.snap

```diff
@@ -13,8 +13,6 @@ finish_describe_definition: describe
 run_start
 run_describe_start: ROOT_DESCRIBE_BLOCK
 run_describe_start: describe
-hook_start: beforeAll
-hook_success: beforeAll
```

Comment:
These hooks shouldn't be removed, don't they?

The describe block doesn't have any skipped or focused tests:
https://github.com/facebook/jest/blob/644dd0d6335651794e2f34ec73e7c82c0bc050d2/packages/jest-circus/src/__tests__/after_all-test.js#L56-L66

Reply:
Oh, I missed the outer hooks hidden by the diff. All good!

Reply:
we should definitely add some intentation to snapshots! :)